src/Framework/Framework/Controls/KnockoutHelper.cs

```diff
@@ -38,6 +34,22 @@ public static class KnockoutHelper
             }
         }
 
+        /// <summary>
+        /// Adds the data-bind attribute to the next HTML element that is being rendered. If in server rendering mode, the binding is also not rendered.
+        /// </summary>
+        /// <param name="renderEvenInServerRenderingMode">When set to true, the binding is rendered even in server rendering mode. By default, the data-bind attribute is only added in client-rendering mode.</param>
+        public static void AddKnockoutDataBind(this IHtmlWriter writer, string name, DotvvmBindableObject control, IValueBinding expression, string? valueUpdate = null, bool renderEvenInServerRenderingMode = false)
+        {
+            if (!control.RenderOnServer || renderEvenInServerRenderingMode)
```

Comment:
I'd just make the `renderEvenInServerRenderingMode` the default. The signature of this method already takes in the `IValueBinding` expression, so I'd expect that it will just render it regardless the RenderSettings. Also we are gradually moving away from the "server rendering disables knockout expression", as users should prefer to use resource bindings for it.